airflow/providers/tableau/operators/tableau_refresh_workbook.py

```diff
@@ -67,18 +68,16 @@ def execute(self, context: dict) -> str:
         with TableauHook(self.site_id, self.tableau_conn_id) as tableau_hook:
             workbook = self._get_workbook_by_name(tableau_hook)
 
-            job_id = self._refresh_workbook(tableau_hook, workbook.id)
-            if self.blocking:
-                from airflow.providers.tableau.sensors.tableau_job_status import TableauJobStatusSensor
+            job_id = TableauOperator(
+                resource='workbooks',
+                method='refresh',
+                resource_id=workbook.id,
+                site_id=self.site_id,
+                tableau_conn_id=self.tableau_conn_id,
+                task_id='refresh_workbook',
+                dag=None,
+            ).execute(context={})
```

Comment:
do we actually actually need a `TableauRefreshWorkbookOperator`?
Your code suggestion means that `TableauRefreshWorkbookOperator` is actually equal to just doing:
```
TableauOperator(resource='workbooks',
                method='refresh',
                resource_id='my_workbook_id',
                site_id='site_id',
                tableau_conn_id='tableau_default',
                task_id='refresh_workbook')
```

So maybe a better way is to deprecate `TableauRefreshWorkbookOperator` ?

Reply:
The main difference is `TableauRefreshWorkbookOperator` uses a find by name and the new uses by id.
Now I add an option to choose what find_by will use, so now the `TableauRefreshWorkbookOperator` is not necessary.
Thanks for the alert